### ISSUE TITLE
"Auto Draft" status for phantom orders

### ISSUE THREAD

**Issue by zwiesman31** [woocommerce/woocommerce-ios#659]
**Describe the bug**
On the app, I see a status of `Auto Draft` for some orders. When I try to check those orders in the WP Admin of my site (`WooCommerce > Orders`) I cannot find them (even if I search for the order number specifically) and have no idea where they're coming from.

I can't produce this on any of my other sites.

**To Reproduce**
1. `Auto Draft` orders display in the app's `Orders` section.
2. Search for them in WP Admin under `WooCommerce > Orders` and cannot find.
3. Search for order numbers specifically and cannot find.

**Screenshots**
![http://cld.wthms.co/IklsW9](http://cld.wthms.co/IklsW9+)
Image link: http://cld.wthms.co/IklsW9

**Expected behavior**
That ghost/phantom orders that don't show anywhere in the `WooCommerce > Orders` section don't display in the app, and also not with the `Auto Draft` status.

**Isolating the problem (mark completed items with an [x]):**
- [x] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [x] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/storefront/).
- [ ] I can reproduce this bug consistently using the steps above.

**Mobile Environment**
Please include:
- Device: iPhone SE
- iOS version: 12.1.3
- WooCommerce iOS version: 1.0.1

**WordPress Environment**
<details>
```
`
### WordPress Environment ###

Home URL: https://allthingsgutenberg.mystagingwebsite.com
Site URL: https://allthingsgutenberg.mystagingwebsite.com
WC Version: 3.5.4
Log Directory Writable: ✔
WP Version: 5.0.3
WP Multisite: –
WP Memory Limit: 256 MB
WP Debug Mode: ✔
WP Cron: ✔
Language: en_US
External object cache: ✔

### Server Environment ###

Server Info: nginx
PHP Version: 7.0.33 - We recommend using PHP version 7.2 or above for greater performance and security. How to update your PHP version
PHP Post Max Size: 100 MB
PHP Time Limit: 300
PHP Max Input Vars: 6144
cURL Version: 7.62.0
OpenSSL/1.0.1t

SUHOSIN Installed: –
MySQL Version: 5.5.5-10.1.17-MariaDB
Max Upload Size: 100 MB
Default Timezone is UTC: ✔
fsockopen/cURL: ✔
SoapClient: ✔
DOMDocument: ✔
GZip: ✔
Multibyte String: ✔
Remote Post: ✔
Remote Get: ✔

### Database ###

WC Database Version: 3.5.4
WC Database Prefix: wp_
MaxMind GeoIP Database: ✔
Total Database Size: 9.94MB
Database Data Size: 8.65MB
Database Index Size: 1.29MB
wp_woocommerce_sessions: Data: 0.02MB + Index: 0.02MB
wp_woocommerce_api_keys: Data: 0.02MB + Index: 0.03MB
wp_woocommerce_attribute_taxonomies: Data: 0.02MB + Index: 0.02MB
wp_woocommerce_downloadable_product_permissions: Data: 0.02MB + Index: 0.05MB
wp_woocommerce_order_items: Data: 0.02MB + Index: 0.02MB
wp_woocommerce_order_itemmeta: Data: 0.06MB + Index: 0.03MB
wp_woocommerce_tax_rates: Data: 0.02MB + Index: 0.06MB
wp_woocommerce_tax_rate_locations: Data: 0.02MB + Index: 0.03MB
wp_woocommerce_shipping_zones: Data: 0.02MB + Index: 0.00MB
wp_woocommerce_shipping_zone_locations: Data: 0.02MB + Index: 0.03MB
wp_woocommerce_shipping_zone_methods: Data: 0.02MB + Index: 0.00MB
wp_woocommerce_payment_tokens: Data: 0.02MB + Index: 0.02MB
wp_woocommerce_payment_tokenmeta: Data: 0.02MB + Index: 0.03MB
wp_woocommerce_log: Data: 0.02MB + Index: 0.02MB
wp_commentmeta: Data: 0.02MB + Index: 0.03MB
wp_comments: Data: 0.02MB + Index: 0.09MB
wp_links: Data: 0.02MB + Index: 0.02MB
wp_options: Data: 3.03MB + Index: 0.05MB
wp_postmeta: Data: 2.13MB + Index: 0.34MB
wp_posts: Data: 0.31MB + Index: 0.06MB
wp_termmeta: Data: 0.02MB + Index: 0.03MB
wp_terms: Data: 0.02MB + Index: 0.03MB
wp_term_relationships: Data: 0.06MB + Index: 0.05MB
wp_term_taxonomy: Data: 0.02MB + Index: 0.03MB
wp_usermeta: Data: 0.02MB + Index: 0.03MB
wp_users: Data: 0.02MB + Index: 0.05MB
wp_wc_booking_relationships: Data: 0.02MB + Index: 0.03MB
wp_wc_download_log: Data: 0.02MB + Index: 0.03MB
wp_wc_gpf_render_cache: Data: 0.02MB + Index: 0.02MB
wp_wc_webhooks: Data: 0.02MB + Index: 0.02MB
wp_woocommerce_gpf_google_taxonomy: Data: 2.52MB + Index: 0.00MB
wp_yoast_seo_links: Data: 0.02MB + Index: 0.02MB
wp_yoast_seo_meta: Data: 0.02MB + Index: 0.00MB

### Post Type Counts ###

attachment: 96
customize_changeset: 1
jetpack_migration: 2
jp_img_sitemap: 3
jp_sitemap: 3
jp_sitemap_master: 3
nav_menu_item: 8
page: 8
post: 1
product: 46
product_variation: 6
revision: 2
shop_coupon: 8
shop_order: 13

### Security ###

Secure connection (HTTPS): ✔
Hide errors from visitors: ✔

### Active Plugins (5) ###

Akismet Anti-Spam: by Automattic – 4.1.1
Jetpack by WordPress.com: by Automattic – 7.0
Storefront Powerpack: by WooCommerce – 1.4.13
WooCommerce Blocks: by Automattic – 1.3.1
WooCommerce: by Automattic – 3.5.4

### Settings ###

API Enabled: –
Force SSL: –
Currency: USD ($)
Currency Position: left
Thousand Separator: ,
Decimal Separator: .
Number of Decimals: 2
Taxonomies: Product Types: booking (booking)
external (external)
grouped (grouped)
simple (simple)
variable (variable)

Taxonomies: Product Visibility: exclude-from-catalog (exclude-from-catalog)
exclude-from-search (exclude-from-search)
featured (featured)
outofstock (outofstock)
rated-1 (rated-1)
rated-2 (rated-2)
rated-3 (rated-3)
rated-4 (rated-4)
rated-5 (rated-5)


### WC Pages ###

Shop base: #8 - /shop/
Cart: #9 - /cart/
Checkout: #10 - /checkout/
My account: #11 - /my-account/
Terms and conditions: ❌ Page not set

### Theme ###

Name: Storefront
Version: 2.4.3
Author URL: https://woocommerce.com/
Child Theme: ❌ – If you are modifying WooCommerce on a parent theme that you did not build personally we recommend using a child theme. See: How to create a child theme
WooCommerce Support: ✔

### Templates ###

Overrides: –
`
```
</details>

**Comment by bummytime** [comment]
Potentially related to the findings in #752:

> ... we see that the orders with that status disappear from the web orders screen but still appear in the mobile app 
> ...
> The reason for this is that /orders endpoint still sends back "deleted" orders (also with the removed order status).

**Comment by melindahelt** [comment]
2409923-zen may be the same. I am confirming if they are seeing the auto-draft label and will update when I hear.

**Comment by mindgraffiti** [comment]
Yep, 2409923-zen is the same problem. We fixed the empty order statuses appearing as auto-draft, but not the problem where deleted orders still appear.

**Comment by mindgraffiti** [comment]
Since this ticket is specifically related to `auto-draft` and that's been addressed, closing this ticket in favor of  #291.
